docs/command-line.md

```diff
@@ -131,6 +131,8 @@ The bundled reporters are:
 
 The JUnit reporter is an xml format that follows the structure of the JUnit XML Report ANT task, as consumed by a number of third-party tools, including Continuous Integration servers such as Jenkins. If not otherwise needed, the standard XML reporter is preferred as this is a streaming reporter, whereas the Junit reporter needs to hold all its results until the end so it can write the overall results into attributes of the root node.
 
+This option may be passed multiple times to use multiple (different) reporters  at the same time. See [Reporters](reporters.md#multiple-reporters) for details.
```

Comment:
Newly written docs should use linebreaks for <80 cols lines.

Also, maybe have the link text be "Multiple Reporters" or "documentation on using multiple reporters"?

Reply:
I am gonna do a larger doc rewrite later.